autocomplete/definitions/global/table/empty.lua

```diff
@@ -1,9 +1,11 @@
 return {
 	type = "function",
-	description = [[Returns true if the table is empty, otherwise, it returns false.]],
+	description = [[Checks if a table is empty. 
+	
+	If `deepCheck==true`, then tables are allowed to have nested subtables, so long as those subtables are empty. e.g., `table.empty({ {}, {} }, true) == true`, while `table.empty({ {}, {} }) == false`.]],
```

Comment:
`deepCheck == true`

Reply:
i made another commit in an attempt to address this and all the suggestions. sorry for messing up the spacing again, i made most of the changes in this PR before i saw your feedback about spacing in the variadic types PR.